### PR TITLE
Add Analog LFCSP-WD-8

### PR DESCRIPTION
**Pull request by jneiva08**
Script for LFCSP WD 8 pins

Datasheet: [link](https://www.analog.com/media/en/technical-documentation/data-sheets/ADP2360.pdf)
Footprint PDF: [link](https://www.analog.com/media/en/package-pcb-resources/package/pkg_pdf/lfcspcp/CP_8_19.pdf)

### CODE REVIEW

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/lfcsp.yaml [hunk 1]
```diff
@@ -1,3 +1,68 @@
...
+
+  thermal_vias:
+    count: [2, 3]
+    drill: 0.3
+    # min_annular_ring: 0.15
+    paste_via_clearance: 0.1
+    EP_num_paste_pads: [2, 3]
+    #paste_between_vias: 1
+    #paste_rings_outside: 1
+    EP_paste_coverage: 0.75
```
Comment (chschlue) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/lfcsp.yaml:52]: Please leave out if there's no particular reason for special coverage.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/lfcsp.yaml [hunk 2]
```diff
@@ -1,3 +1,68 @@
...
+    minimum: 2.34
+  # EP_paste_coverage: 0.65
+  EP_num_paste_pads: [2, 3]
+  #heel_reduction: 0.05 #for relatively large EP pads (increase clearance)
+
+  thermal_vias:
+    count: [2, 3]
+    drill: 0.3
+    # min_annular_ring: 0.15
+    paste_via_clearance: 0.1
```
Comment (chschlue) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/lfcsp.yaml:48]: Unnecessary if paste_avoid_via is false.

**Comment by chschlue** [comment]
https://github.com/KiCad/kicad-footprints/pull/2253

**Comment by jneiva08** [comment]
@chschlue, To confirm, need to comment line 48 and 52 rightt?

**Comment by chschlue** [comment]
Correct. Or remove entirely.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/lfcsp.yaml (hunk 2 above)
Comment (jneiva08) [reply scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/lfcsp.yaml]: done

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/lfcsp.yaml (hunk 1 above)
Comment (jneiva08) [reply scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/lfcsp.yaml]: done

**Comment by evanshultz** [comment]
Thank you!

**Comment by jneiva08** [comment]
Thanks for your time

**Pull request merged**